src/modules/comments.js

```diff
@@ -0,0 +1,74 @@
+const fillDetails = async (id) => {
+  const baseApi = 'https://api.tvmaze.com/shows/';
+  const list = await fetch(`${baseApi}${id}`).then((response) => response.json());
+  return list;
+};
+
+const addClosePopUpevent = () => {
+  const closeBtn = document.querySelector('.mclose-btn');
+  closeBtn.addEventListener('click', () => {
+    const popupModal = document.querySelector('.modal-wraper');
+    popupModal.style.display = 'none';
+  });
+};
+
+export const createPopUpDetails = (details) => {
+  const projectModal = document.createElement('div');
+  projectModal.className = 'movie-pop-up';
+
+  projectModal.innerHTML = `
+  <div class="mclose-btn">
+  <span class="close-span">X</span>  
+  </div>
+  
+  <div class= 'modal-image'>
+  <img
+  src = "${details.url}";
+  alt = "cover image"
+  id = "modal-cover"
+  />
```

Comment:
Kindly recheck this part it does not display the image as it should
![image](https://user-images.githubusercontent.com/67240581/203983004-e6f4275f-55f1-4507-8090-9e8ba59a835f.png)

Reply:
Hi @ernestkorir 
I have added the movie image and it is now displaying.
Please check it out.